SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1525,6 +1574,19 @@ int32 USpatialNetDriver::ServerReplicateActors(float DeltaSeconds)
 	// Build the consider list (actors that are ready to replicate)
 	ServerReplicateActors_BuildConsiderList(ConsiderList, ServerTickTime);
 
+	
+	const ASpatialWorldSettings* SpatialWorldSettings = Cast<ASpatialWorldSettings>(WorldSettings);
+	const bool bIsMultiWorkerEnabled = SpatialWorldSettings != nullptr && SpatialWorldSettings->IsMultiWorkerEnabled();
+
+	FSpatialLoadBalancingHandler MigrationHandler(this);
+	FSpatialNetDriverLoadBalancingContext LoadBalancingContext(this, ConsiderList);
+
+	if (bIsMultiWorkerEnabled)
+	{
+		MigrationHandler.HandleLoadBalancing(LoadBalancingContext);
+		LoadBalancingContext.UpdateWithAdditionalActors();
```

Comment:
Why is this a separate call vs. calling it within `HandleLoadBalancing`?

Reply:
Because I changed the pattern. It was possible with CRTP, using method hiding. Not possible anymore with Traits pattern.

Reply:
Also, I need to wait till the end of the processing to add them to the consider list, because I could add actors, but remove them later if I discover they are already in the list.